unittest/commands/CMakeLists.txt

```diff
@@ -8,6 +8,13 @@ target_link_libraries(test_lattice_region PRIVATE lammps GTest::GMock GTest::GTe
 add_test(NAME LatticeRegion COMMAND test_lattice_region WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 add_executable(test_kim_commands test_kim_commands.cpp)
+if(KIM_EXTRA_UNITTESTS)
+  if(CURL_FOUND)
+    target_compile_definitions(test_kim_commands PRIVATE -DKIM_EXTRA_UNITTESTS)
+  else()
+    message(STATUS "CURL not found. Enabling KIM extra unit tests requires to have libcurl installed.")
```

Comment:
shouldn't this be a `FATAL_ERROR` instead of a `STATUS` message?

Reply:
thanks, yes it makes sense. I updated the mode to `FATAL_ERROR`.